packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -111,32 +117,30 @@ describe('ui:create:atomic', () => {
       'src/html.d.ts',
       'src/components.d.ts',
       'src/style/index.css',
-      'src/style/layout.css',
-      'src/style/theme.css',
       'src/pages/index.html',
       'src/components/results-manager/results-manager.tsx',
-      'src/components/results-manager/default.html',
+      'src/components/results-manager/template-1.html',
       'src/components/sample-component/sample-component.tsx',
       'src/components/sample-component/sample-component.css',
       'src/components/sample-result-component/sample-result-component.tsx',
       'src/components/sample-result-component/sample-result-component.css',
     ];
 
-    createdFilesPaths.forEach((path) =>
-      expect(projectFileExist(path)).toBe(true)
-    );
-  });
+    createdFilesPaths.forEach((path) => {
+      it(`should create the "${path}" file or directory`, () =>
+        expect(projectFileExist(path)).toBe(true));
+    });
```

Comment:
This no, instead https://jestjs.io/docs/api#testeachtablename-fn-timeout or https://jestjs.io/docs/api#describeeachtablename-fn-timeout

Same for all foreaches